battleship/Form1.cs

```diff
@@ -12,9 +12,26 @@ namespace battleship
 {
     public partial class Form1 : Form
     {
+        WMPLib.WindowsMediaPlayer musicPlayer = new WMPLib.WindowsMediaPlayer();
         public Form1()
         {
             InitializeComponent();
         }
+
+        private void Form1_Load(object sender, EventArgs e)
+        {
+            musicPlayer.URL = (@"StartMusic.mp3");
+            musicPlayer.settings.volume = 15;
+            musicPlayer.controls.play();
+        }
+
+        private void beginButton_Click(object sender, EventArgs e)
+        {
+            startScreen.Enabled = false;
+            startScreen.Visible = false;
+            beginButton.Enabled = false;
+            beginButton.Visible = false;
+            musicPlayer.controls.stop();
```

Comment:
Yeah that would be easy to do. Right now it's just playing music during the
start screen and then it stops when the game begins. If you find an mp3
file you like I can put it in
On Nov 10, 2015 11:37 AM, "Ontrox" <notifications@github.com> wrote:

> In battleship/Form1.cs
> <https://github.com/ninoaguilar/Battleship/pull/23#discussion_r44445518>:
>
> >          public Form1()
> >          {
> >              InitializeComponent();
> >          }
> > +
> > +        private void Form1_Load(object sender, EventArgs e)
> > +        {
> > +            musicPlayer.URL = (@"StartMusic.mp3");
> > +            musicPlayer.settings.volume = 15;
> > +            musicPlayer.controls.play();
> > +        }
> > +
> > +        private void beginButton_Click(object sender, EventArgs e)
> > +        {
> > +            startScreen.Enabled = false;
> > +            startScreen.Visible = false;
> > +            beginButton.Enabled = false;
> > +            beginButton.Visible = false;
> > +            musicPlayer.controls.stop();
>
> Just when the player clicks the begin button, to have an audio transition
> to the beginning of
> the game.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ninoaguilar/Battleship/pull/23/files#r44445518>.
>